### PR TITLE
Fix mismatched Retrofit artifact versions

### PR DESCRIPTION
**Pull request by NightlyNexus**

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -61,7 +61,7 @@ dependencies {
 
   //networking
   compile 'com.squareup.retrofit2:retrofit:2.1.0'
-  compile 'com.squareup.retrofit2:converter-gson:2.0.0'
+  compile 'com.squareup.retrofit2:converter-gson:2.1.0'
```
Comment (es0329) [inline app/build.gradle]: Good catch! I understand this as the textbook case for using variables to manage dependency version numbers. What do you think of...

```
project.ext.set("retrofitVersion", "2.1.0")

dependencies {
  ...
  compile "com.squareup.retrofit2:retrofit:${retrofitVersion}"
  compile "com.squareup.retrofit2:converter-gson:${retrofitVersion}"
  ...
}
```

Comment (NightlyNexus) [inline app/build.gradle]: yeah, we could, if you prefer.

File: app/build.gradle [hunk 2]
```diff
@@ -31,6 +31,12 @@ android {
   }
 }
 
+ext {
+  okHttpVersion = '3.5.0'
```
Comment (NightlyNexus) [inline app/build.gradle]: Consider updating to 3.6.0.

File: app/build.gradle [hunk 3]
```diff
@@ -31,6 +31,12 @@ android {
...
+  retrofitVersion = '2.1.0'
+  supportLibraryVersion = '25.0.1'
```
Comment (NightlyNexus) [inline app/build.gradle]: Consider updating to 25.1.0.

File: app/build.gradle [hunk 4]
```diff
@@ -57,12 +63,12 @@ dependencies {
...
-  androidTestCompile 'com.android.support:support-annotations:25.0.1'
+  androidTestCompile "com.android.support:support-annotations:${supportLibraryVersion}"
 
   //networking
-  compile 'com.squareup.retrofit2:retrofit:2.1.0'
-  compile 'com.squareup.retrofit2:converter-gson:2.0.0'
-  compile 'com.squareup.okhttp3:logging-interceptor:3.5.0'
+  compile "com.squareup.retrofit2:retrofit:${retrofitVersion}"
+  compile "com.squareup.retrofit2:converter-gson:${retrofitVersion}"
+  compile "com.squareup.okhttp3:logging-interceptor:${okHttpVersion}"
```
Comment (NightlyNexus) [inline app/build.gradle]: Consider specifying `compile "com.squareup.okhttp3:okhttp:${rootProject.okhttpVersion}"`. OkHttp gets updated more frequently than Retrofit, so this is currently bringing in an old version of OkHttp.

File: app/build.gradle [hunk 5]
```diff
@@ -40,9 +46,9 @@ dependencies {
   compile 'org.jetbrains.anko:anko-common:0.8.3'
 
   //android
-  compile 'com.android.support:appcompat-v7:25.0.1'
-  compile 'com.android.support:design:25.0.1'
-  compile 'com.android.support:recyclerview-v7:25.0.1'
+  compile "com.android.support:appcompat-v7:${supportLibraryVersion}"
+  compile "com.android.support:design:${supportLibraryVersion}"
+  compile "com.android.support:recyclerview-v7:${supportLibraryVersion}"
   compile 'com.google.android.gms:play-services-location:10.0.0'
```
Comment (NightlyNexus) [inline app/build.gradle]: Consider 10.0.1?

**Pull request merged**